docs/wiki/User Guide/Configurations.md

```diff
@@ -112,6 +112,8 @@ The following configurations are inherited from the open source Kafka client con
 | metric.anomaly.analyzer.metrics               | String  | N         | ""                                                                                                                                                                                                                                                                                                                                                                                                                     | The metric ids that the metric anomaly detector should detect if they are violated.                                                                                                                                                                                                                                                        |
 | failed.brokers.zk.path                        | String  | N         | /CruiseControlBrokerList                                                                                                                                                                                                                                                                                                                                                                                               | The zk path to store the failed broker list. This is to persist the broker failure time in case Cruise Control failed and restarted when some brokers are down.                                                                                                                                                                                                                                                     |
+| fixable.failed.broker.count.threshold         | Integer | N         | 5                                                                                                                                                                                                                                                                                                                                                                                               |The upper boundary of concurrently failed broker count that are taken as fixable.                                                                                                                                                                                                                                                     |
```

Comment:
Based on the current defaults, `fixable.failed.broker.count.threshold` will be the limiting factor in any cluster larger than `12 brokers`. Should we adjust these limits such that they map to the equal broker count for a medium cluster size -- e.g. a cluster of `25 brokers` leads to `fixable.failed.broker.count.threshold` default value of `10`?